src/templates/article.tsx

```diff
@@ -421,6 +441,29 @@ const MainArticleTitle = styled.div`
   }
 `
 
+const MainArticleNav = styled.ul`
+  list-style: none;
+  margin-block: 5rem;
+  padding: 0;
+  display: grid;
+  gap: 1rem;
+  grid-template: 'left right' 1fr/50%;
```

Comment:
```suggestion
  grid-template: 'left right' 1fr/1fr 1fr;
```

@yuw27b @versionfive さんに指摘をいただいたのですが、左右のリンクで16pxの幅の違いがありました。
50%はあくまで親コンテナとの比率でgapの幅が考慮されないためだと思われます。
収まる範囲で同じ幅にするにはsuggetionのような対応が必要になるかと思います。
`1fr 1fr` ではなく、`50% 50%` だと16px分親コンテナからはみ出ます。手元で確認して修正お願いします🙏

Reply:
@tomof すみません、修正しました！現在は同じ幅になっているかと思います。